tests/golden/pr2302.tsv

```diff
@@ -1,4 +1,3 @@
-# comment						
```

Comment:
This test was added for testing https://github.com/saulpw/visidata/pull/2302 @midichef. Is the behaviour you addressed in that PR now broken?

Reply:
Oh, nice catch. I didn't look into the history of this test. The behavior in that PR is still functional. But it is obsolete after https://github.com/saulpw/visidata/commit/6fff636f33f2ffeaac2e74359f8d199c2358c960.
I'll open a PR removing the remaining uses of `regex_skip` from `tsv.py`.

Reply:
On closer inspection, the remaining uses of `regex_skip` in tsv.py look to be intentional, so I'll leave it as is.